PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -78,28 +78,14 @@ function configureToolboxSettings(settings) {
     var fileCollection = folder.SubFolders;
 
 
-    var maxMajor = 0,
-        maxMinor = 0,
-        maxPatch = 0,
-        maxVersionFolder = "";
+    var maxVersionFolder = "0";
```

Comment:
Please replace `"0"` with `"0.0.0"`.

---

We no longer compare individual major/minor/patch, but whole versions.

Reply:
the versioncompare function automatically sees this as 0.0.0, so I don't think it's necessary

Reply:
OK.